src/assets/styles/components/_filters.css

```diff
@@ -0,0 +1,111 @@
+
+[class*="--hide"] {
+    display: none;
+}
+
+.filters {
+    margin-block-end: 1.75rem;
+}
+
+.filters inclusive-disclosure {
+    padding-inline: 0;
+}
+
+.filters inclusive-disclosure * ul {
+    padding-inline: 0;
+}
+
+.filters fieldset {
+    border: unset;
+    margin: unset;
+    padding: unset;
+}
+
+.filter-tags {
+    border-top: 0.125rem solid var(--fl-fgColor, var(--color-indigo-700));
+    margin-block-end: 2.8125rem;
+    margin-block-start: calc(1em + 1.25rem);
+}
+
+.filter-tags > h4 {
+    display: none;
+    margin-block-end: 1.5rem;
+    margin-block-start: 1.0625rem;
+}
+
+.filter-tag {
+    background-color: transparent;
+    border: 0.0625rem solid var(--fl-fgColor, var(--color-indigo-700));
+    border-radius: 18.75rem;
+    color: var(--fl-fgColor, var(--color-indigo-700));
+    font-family: var(--family-sans-serif);
+    font-size: calc(var(--step--1) * var(--fl-enhance-font-size-factor, 1));
+    font-weight: var(--fl-enhance-font-weight, var(--font-weight-medium));
+    margin-block-end: 1rem;
+    margin-inline-end: 1rem;
+    padding-block: 0.55rem;
+    padding-inline: 2.7rem;
+    position: relative;
+
+    &:hover,
+    &:focus {
+        box-shadow:
+            0 0 0 0.125rem var(--fl-linkBgColor, var(--color-white)),
+            0 0 0 0.25rem var(--fl-linkFgColor, var(--color-indigo-700));
+        outline: 1px solid transparent;
+    }
+
+    &:active,
+    &:active:hover {
+        background-color: var(--fl-linkBgColor, var(--color-indigo-200));
+        box-shadow: inset 0 0 0 0.125rem
+            var(--fl-linkFgColor, var(--color-indigo-700));
+        color: var(--fl-linkFgColor, var(--color-indigo-700));
+    }
+}
+
+.filter-tag::before {
```

Comment:
Consider combining shared rules for `.filter-tag::before` and `.filter-tag::after` into a `.filter-tag::before, .filter-tag::after` rule and then only placing unique rules in these individual selectors.